packages/components/src/organisms/ShippingSimulation/ShippingSimulation.tsx

```diff
@@ -0,0 +1,184 @@
+import type { HTMLAttributes } from 'react'
+import type { PriceFormatter } from '../../atoms/Price/Price'
+
+import React from 'react'
+
+import { ArrowSquareOut } from '../../assets'
+
+import {
+  Icon,
+  Link,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  InputField,
+  Price,
+  LinkProps,
+  LinkElementType,
+} from '../..'
+
+interface ShippingSla {
+  /**
+   * ShippingSLA carrier.
+   */
+  carrier: string
+  /**
+   * ShippingSLA localized shipping estimate.
+   */
+  localizedEstimates: string
+  /**
+   * ShippingSLA price.
+   */
+  price: number
+}
+
+export interface ShippingSimulationProps
+  extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Formatter function that transforms the raw price value and render the result.
+   */
+  formatter: PriceFormatter
+  /**
+   * The Shipping Suggestions Section's title.
+   */
+  title?: string
+  /**
+   * The text displayed to Shipping Simulation input text.
+   */
+  inputLabel?: string
+  /**
+   * The text displayed to Shipping Simulation input text.
+   */
+  optionsLabel?: string
+  /**
+   * Props for the link for i don't know my postal code.
+   */
+  idkPostalCodeLinkProps?: Partial<LinkProps<LinkElementType>>
```

Comment:
hahha, I thought this was a mistake 🤣

Reply:
hahaha, it is right, because it is the props for link "I don't know my postal code", but we can think about a better description haha